db/migrate/20200627060227_add_name_to_classrooms.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToClassrooms < ActiveRecord::Migration[6.0]
```

Comment:
please merge migrations that change things in the same model
i.e.: we don't need to separate migrations to add a column, but create a migration with a different name and add the two change actions in there